front-end/src/PreferenceModal.jsx

```diff
@@ -0,0 +1,188 @@
+                                25%
+                            </SliderMark>
+                            <SliderMark value={50} {...labelStyles}>
+                                50%
+                            </SliderMark>
+                            <SliderMark value={75} {...labelStyles}>
+                                75%
+                            </SliderMark>
+                            <SliderTrack>
+                                <SliderFilledTrack sx={{ bgColor: 'rgb(83, 41, 140)' }} />
+                            </SliderTrack>
+                            <Tooltip
+                                hasArrow
+                                bg='rgb(83, 41, 140)'
+                                color='white'
+                                placement='top'
+                                isOpen={showMoodValue}
+                                label={`${moodValue}%`}
+                            >
+                                <SliderThumb />
+                            </Tooltip>
+                        </Slider>
+                        <Text color='gray.700' fontSize='sm' mt='1.5em'>How important is consistent mood in your stitch? 100% = consistent 🙂.</Text>
+                    </Box>
+                    <Text color='black' as='b'>Danceability</Text>
+                    <Box p={2} pt={7} mb='1em'>
+                        <Slider aria-label='slider-ex-6'
+                            value={danceValue}
+                            onChange={(val) => setDanceValue(val)}
+                            onMouseEnter={() => setShowDance(true)}
+                            onMouseLeave={() => setShowDance(false)}>
+                            <SliderMark value={25} {...labelStyles}>
+                                25%
+                            </SliderMark>
+                            <SliderMark value={50} {...labelStyles}>
+                                50%
+                            </SliderMark>
+                            <SliderMark value={75} {...labelStyles}>
+                                75%
+                            </SliderMark>
+                            <SliderTrack>
+                                <SliderFilledTrack sx={{ bgColor: 'rgb(83, 41, 140)' }} />
+                            </SliderTrack>
+                            <Tooltip
+                                hasArrow
+                                bg='rgb(83, 41, 140)'
+                                color='white'
+                                placement='top'
+                                isOpen={showDanceValue}
+                                label={`${danceValue}%`}
+                            >
+                                <SliderThumb />
+                            </Tooltip>
+                        </Slider>
+                        <Text color='gray.700' fontSize='sm' mt='1.5em'>How important is dancing in your stitch? 100% = dance party 🕺.</Text>
+                    </Box>
```

Comment:
In a further iteration, I think each of these sliders could also be reused - and be the same component basically - instead of having 4 very similar sliders with string changes and associated values.